common/api/core-backend.api.md

```diff
@@ -669,11 +669,17 @@ export class Category extends DefinitionElement {
     protected constructor(props: CategoryProps, iModel: IModelDb);
     // (undocumented)
     static get className(): string;
+    // @beta (undocumented)
```

Comment:
Beta APIs need documentation. It looks like all of the ones added in this PR are undocumented.